server/controllers/Accounts.js

```diff
@@ -115,12 +115,12 @@ class Accounts {
    */
   static async debitAccount(req, res) {
     const { accountNumber } = req.params;
-    const { amount, accountnumber } = req.body;
+    const { amount } = req.body;
     const { email } = req.user;
     try {
       const staff = await staffs.findByEmail('*', [email]);
       const s = `${staff.firstname} ${staff.lastname}`;
-      const account = await accounts.findByAccountNumber('*', [accountnumber]);
+      const account = await accounts.findByAccountNumber('*', [accountNumber]);
```

Comment:
what's the purpose of `*`  in `accounts.findByAccountNumber('*', [accountNumber])`

Reply:
Should I add it as a comment or explain here. if here === `true`, I used a modular function that just helps me select columns from my database, * means I'm calling all the key-value pairs in the database.